src/content/docs/data-apis/understand-data/metric-data/cumulative-metrics.mdx

```diff
@@ -29,7 +29,11 @@ For more information, see [Prometheus remote write integration](/docs/infrastruc
 
 ## OpenTelemetry support [#otel-support]
 
-For more information, see [OpenTelemetry metrics: Best practices](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-metrics/).
+<Callout variant="important">
```

Comment:
i must have screwed something up because it's not going to be in a preview. I must have lost some edits to that doc or something. Sorry about confusion.